src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -176,7 +176,6 @@ function ReimbursementAccountPage({route, policy}: ReimbursementAccountPageProps
      which acts similarly to `componentDidUpdate` when the `reimbursementAccount` dependency changes.
      */
     const [hasACHDataBeenLoaded, setHasACHDataBeenLoaded] = useState(reimbursementAccount !== CONST.REIMBURSEMENT_ACCOUNT.DEFAULT_DATA && isPreviousPolicy);
-    const [shouldShowContinueSetupButton, setShouldShowContinueSetupButton] = useState(getShouldShowContinueSetupButtonInitialValue());
```

Comment:
We are setting the default value as getShouldShowContinueSetupButtonInitialValue(). But this value only be set in the first rendering. If the page is rendered, getShouldShowContinueSetupButtonInitialValue isn't triggered again -->  shouldShowContinueSetupButton isn't updated 

So I create a new useEffect to update this state

Reply:
cc @koko57